src/database/migrations/1745051642191-create-college-table.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class CreateCollegeTable1745051642191 
+implements MigrationInterface {
+    name = 'CreateCollegeTable1745051642191'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`CREATE TABLE "college" (
+            "id" SERIAL PRIMARY KEY, 
+            "name" VARCHAR(255) NOT NULL, 
+            "uniform" VARCHAR(255), 
+            "provost" VARCHAR(255), 
```

Comment:
Provost as how

And again provost can't be string. Ideally it should be a relation with another table. Leave it out for now. You'll update the table later to do this